pkg/sql/crdb_internal.go

```diff
@@ -337,6 +338,57 @@ CREATE TABLE crdb_internal.tables (
 	},
 }
 
+var crdbInternalTablesTableLastStats = virtualSchemaTable{
+	comment: "the latest stats for all tables accessible by current user in current database (KV scan)",
+	schema: `
+CREATE TABLE crdb_internal.table_row_stats (
+  table_id         INT         NOT NULL,
+  table_name       STRING      NOT NULL,
+  row_count        INT         NOT NULL
```

Comment:
I'd name this `estimated_row_count` or something like that, just so it's clear that it is an estimate.

Reply:
Renamed.